Presentation.Web/app/services/exportGridToExcelService.ts

```diff
@@ -9,33 +10,17 @@
 
         constructor(private readonly needsWidthFixService: NeedsWidthFix) { }
 
-        getExcel(e: IKendoGridExcelExportEvent<any>, _: ILoDashWithMixins, timeout: ng.ITimeoutService, kendoGrid: IKendoGrid<any>) {
+        getExcel(e: IKendoGridExcelExportEvent<any>, _: ILoDashWithMixins, timeout: ng.ITimeoutService, kendoGrid: IKendoGrid<any>, config?: IExcelConfig) {
             const columns = e.sender.columns;
 
             if (!this.exportFlag) {
                 e.preventDefault();
-                _.forEach(columns, (column, i) => {
-                    if (!column.hidden) {
-                        this.columnsToShow.push({ columnId: column.persistId });
-                        return;
-                    }
-                    if (column.parentId === undefined)
-                        return;
-                    //look for a parent column and check if any was found, and if any parent column should be visible
-                    var columnsWithMatchingParentId = columns.filter(x => x.persistId === column.parentId);
-                    if (columnsWithMatchingParentId.length !== 1) {
-                        console.error("Column has multiple columns with matching parentId");
-                        return;
-                    }
-                    const parentColumn = columnsWithMatchingParentId[0];
-                    if (parentColumn.hidden)
-                        return;
-                    
-                    var parentIndex = columns.indexOf(parentColumn);
+                var onlyVisibleColumns = false;
+                if (config !== undefined && config.onlyVisibleColumns !== undefined) {
+                    onlyVisibleColumns = config.onlyVisibleColumns;
```

Comment:
can be simplified into 
`onlyVisibleColumns = config?.onlyVisibleColumns === true`